### ISSUE TITLE
Communicate the basics of using SQL querying to users

### ISSUE THREAD

**Issue by scw** [genegis/genegis#30]
Because most users of the tool will have had limited exposure to data manipulation beyond Excel, and perhaps other data frame based languages such as R, we'll want to give them an easy introduction to the basics of querying (or at least, the Esri take on the WHERE clause) to get them up to speed. This will simplify the steps they carry out, in tools such as the 'select by attributes' geoprocessing tool that Dori had originally developed. 

One starting point is the existing ArcGIS 10.1 documentation on the topic:
  http://resources.arcgis.com/en/help/main/10.1/index.html#//00s50000002t000000

**Comment by deepseadawn** [comment]
I agree on the easy introduction. I think it will also be very interesting to see how this is received once we get a nice core of users of geneGIS, as I suspect there will be a continuum along "generational lines." That is the grad students will likely be "digital natives" and perhaps even well versed with ArcGIS 10 and R already, while the more senior scientists will most certainly appreciate the help. Another factor here that I think is excellent and crucial, is the drilling down into the Esri documentation that Shaun has already done, to get right to the specific help page that is needed. Without this, people would still be lost as our documentation pages, while extremely rich, are now a behemoth, and the search functions are not always going to achieve the desired result.
Thanks!
